src/include/dav1d/dav1d.rs

```diff
@@ -175,11 +176,14 @@ impl TryFrom<Dav1dSettings> for Rav1dSettings {
             decode_frame_type,
             reserved: _,
         } = value;
+        validate_input!((n_threads <= 256, Rav1dError::EINVAL))?;
+        validate_input!((max_frame_delay <= 256, Rav1dError::EINVAL))?;
+        validate_input!((operating_point <= 31, Rav1dError::EINVAL))?;
```

Comment:
These should keep their `>= 0`s so that the full check is in `validate_input!` and then the below checks can use `.try_into().unwrap()`, which will optimize out.